tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -606,6 +607,8 @@ private void OnMvcBeforeAction(object arg)
                 if (span is not null)
                 {
                     CurrentSecurity.CheckPathParamsFromAction(httpContext, span, typedArg.ActionDescriptor?.Parameters, typedArg.RouteData.Values);
+
+                    AddCodeOriginTags(typedArg, span);
```

Comment:
do we want to place this call above `CheckPathParamsFromAction`? This function could throw `BlockException`, in which case I think it will very useful to see Span Origin for those cases too.

Reply:
Fixed in 5792875cc54444656eb03006177ac1fe4166d3f7